### PR TITLE
Footer stuff

### PR DESCRIPTION
**Pull request by wdwatkins**
Not ready to go, just wanted to get stuff up so its visible

### CODE REVIEW

File: inst/testviz/scripts/fetch/mayflyNymph.R [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #' Could also accept the data.info arg but won't because we don't use it.
 #' Intentionally switching up the arg order just to show that it can be done
 #' (though it's not necessarily a great idea)
-fetchData.mayflyNymph <- function(outfile, viz.id, ...) {
+fetch.mayflyNymph <- function(outfile, viz.id, ...) {
```
Comment (jiwalker-usgs) [inline inst/testviz/scripts/fetch/mayflyNymph.R]: The arguments need to change here as well `fetch.mayflyNymph <- function(viz)`

**Comment by jiwalker-usgs** [comment]
@wdwatkins I merged @lindsaycarr's PR before this one, so there are a couple conflicts.  Would you be able to resolve these?

**Comment by jiwalker-usgs** [comment]
Otherwise, I'll work with @mwernimont to figure out our style issues.

**Comment by wdwatkins** [comment]
@jiwalker-usgs sure.  I deleted the footer stuff from main.css so that footer looks normal now, I'll push it up in a sec

**Comment by wdwatkins** [comment]
Nevermind, that css is in the example viz repo

**Comment by wdwatkins** [comment]
Merging so we can use this on the matthew viz https://github.com/USGS-VIZLAB/hurricane-matthew/issues/54, but test setup needs work.  See #83

**Pull request merged**